yarn.lock

```diff
@@ -188,11 +188,7 @@
   version "3.0.0"
   resolved "https://registry.yarnpkg.com/@mapbox/whoots-js/-/whoots-js-3.0.0.tgz#c1de4293081424da3ac30c23afa850af1019bb54"
 
-"@types/node@*":
```

Comment:
Are there actual dependency updates in here or is this an accidental change (different version of yarn maybe?)